### ISSUE TITLE
The react/jsx-space-before-closing rule is deprecated. Please use the react/jsx-tag-spacing rule with the "beforeSelfClosing" option instead

### ISSUE THREAD

**Issue by Flet** [feross/eslint-config-standard-jsx#23]
Reported by @donovanhiland in feross/eslint-config-standard#83

**Comment by modestfake** [comment]
I have the latest version of `eslint-config-standard-jsx`, but I still see this warning.
I have this line in `node_modules/eslint-config-standard-jsx/eslintrc.json`:
```
"react/jsx-space-before-closing": "error"
```
but not this:
```
"react/jsx-tag-spacing": ["error", { "beforeSelfClosing": "always" }]
```

**Comment by i-break-codes** [comment]
Any fix yet? I haven't specified anything in the rules, and still, I see this warning.

**Comment by jeffs** [comment]
Why was this closed?  Is any workaround available?

**Comment by donovanhiland** [comment]
This package is used in `eslint-config-standard-react` which doesn't have the updated dependency version that includes this fix.

Until that happens you can add:
```
"rules": {
  "react/jsx-tag-spacing": ["error", { "beforeSelfClosing": "always" }]
}
```
to your `.eslintrc` which will suppress the error

**Comment by provDeepakAhuja** [comment]
Hi,
I am getting this error still.. even after adding the rule.. any other pointer.. 


The react/jsx-space-before-closing rule is deprecated. Please use the react/jsx-tag-spacing rule with the "beforeSelfClosing" option instead.
Cannot read property 'range' of null


Below is my .eslintrc
{
  "extends": ["standard", "standard-react"],
  "globals": {
    "__DEVELOPMENT__": true,
    "__CLIENT__": true,
    "__DEVTOOLS__": true,
    "__DISABLE_SSR__": true,
    "fetch": false,
    "fail": false,
    "test": false,
    "describe": false,
    "expect": false,
    "jest": false,
    "afterAll": false,
    "beforeAll": false,
    "afterEach": false,
    "beforeEach": false
  },
  "rules": {
    "indent" : ["error", 2, { "ignoredNodes": ["JSXElement *"] }],
    "object-curly-spacing": ["error", "always", { "objectsInObjects": false }],
    "standard/object-curly-even-spacing": "off",
    "jsx-quotes": ["error", "prefer-double"],
    "react/jsx-tag-spacing": ["error", { "beforeSelfClosing": "always" }],
    "react/jsx-curly-spacing": [2, "never", {"allowMultiline": true}],
    "react/jsx-indent-props": ["error", 4],
    "no-unused-expressions": "off"
  },
  "settings": {
    "react": {
      "version": "latest"
    }
  },
  "parser": "babel-eslint"
}
